Source/CocoaMQTT.swift

```diff
@@ -498,10 +501,18 @@ extension CocoaMQTT: GCDAsyncSocketDelegate {
         autoReconnTimer = nil
         if disconnectExpectedly {
             connState = .initial
-        } else if autoReconnect && autoReconnectTimeInterval > 0 {
+        } else if autoReconnect {
             autoReconnTimer = CocoaMQTTTimer.every(Double(autoReconnectTimeInterval), { [weak self] in
-                printInfo("Try reconnect")
-                _ = self?.connect()
+                guard let weakSelf = self else {
```

Comment:
This is not a `weakSelf`, instead, when creating this let it turns into a `strongSelf`.
What's the reasoning for this naming convention? 
I have at least seen it once other time in the codebase

Reply:
Hi @halonsoluis, sorry for wrong convention, I will be updating it. Thanks. https://github.com/apple/swift-evolution/blob/master/proposals/0079-upgrade-self-from-weak-to-strong.md is a swift proposal, I meant to use it.